frontend/src/store/config.js

```diff
@@ -230,19 +230,19 @@ export const useConfigStore = defineStore('config', () => {
   })
 
   const controlPlaneHighAvailabilityHelp = computed(() => {
-    return state.value?.shootDefaults.controlPlaneHighAvailabilityHelp ?? state.value?.controlPlaneHighAvailabilityHelp
+    return state.value?.shootDefaults?.controlPlaneHighAvailabilityHelp ?? state.value?.controlPlaneHighAvailabilityHelp
```

Comment:
we should introduce a `shootDefaults` computed that defaults to `{} ` to reduce optional chaining.

Reply:
ping @benedikt-haug

Reply:
I saw that you added a default to the helm chart. However, the better approach would be to add a computed property like this:
```js
const shootDefaults = computed(() => {
  return state.value?.shootDefaults ?? {}
})
```
This way we have the defaulting also in case of operator deployment. Also, this shortens all your other computed logic as you can do like
```js
const defaultHibernationSchedule = computed(() => {
  return shootDefaults.hibernationSchedule ?? state.value?.defaultHibernationSchedule
})
```
for all the computed getters.